test/unit/core/single/sessions.test.js

```diff
@@ -484,10 +484,15 @@ describe('Sessions (Single)', function() {
         // Execute next
         cursor._next(function(err) {
           expect(err).to.not.exist;
+          expect(session.id).to.exist;
 
           cursor.kill(err => {
             expect(err).to.not.exist;
-            commands.forEach(command => expect(command.lsid).to.eql(session.id));
+
+            commands.forEach(command => {
+              if (command.killCursors) return;
```

Comment:
Is this a new change with versioned API?

Reply:
This test (which has been moved to `disabled/` in 4.0) started failing after the NODE-3183 backport, because `command.killCursors` doesn't have an lsid. I'll do a bit more investigation into what is happening with this test.

Reply:
Turns out this was a bug I introduced, fixed now! 👍